src/main/java/org/reaktivity/nukleus/tls/internal/TlsNukleus.java

```diff
@@ -70,17 +72,18 @@
 
     private final TlsConfiguration config;
     private final Map<RouteKind, MessagePredicate> routeHandlers;
-    private final Map<String, MutableInteger> routesByStore;
-    private final Long2ObjectHashMap<String> storesByRouteId;
-    private final Map<String, SSLContext> contextsByStore;
+    private final Map<Long, String> storesByRouteId;
```

Comment:
Use `Long2ObjectHashMap`

Reply:
Didn't use Long2ObjectHashMap since the values can be null.